resources/repo.rb

```diff
@@ -1,28 +1,19 @@
-property :repo_name, String, desired_state: false, identity: true, name_attribute: true
-property :repo_type, String, desired_state: false, identity: true, default: 'maven2-hosted'
+property :repo_name, String, name_attribute: true
+property :repo_type, String, default: 'maven2-hosted'
 property :attributes, Hash, default: lazy { ::Mash.new } # Not mandatory but strongly recommended in the generic case.
 property :online, [true, false], default: true
-property :api_endpoint, String, desired_state: false, identity: true,
-                                default: lazy { node['nexus3']['api']['endpoint'] }
-property :api_username, String, desired_state: false, identity: true,
-                                default: lazy { node['nexus3']['api']['username'] }
-property :api_password, String, desired_state: false, identity: true, sensitive: true,
+property :api_endpoint, String, desired_state: false, default: lazy { node['nexus3']['api']['endpoint'] }
+property :api_username, String, desired_state: false, default: lazy { node['nexus3']['api']['username'] }
+property :api_password, String, desired_state: false, sensitive: true,
```

Comment:
should be `identity: true` otherwise you don't have access to it in `load_current_value`

Reply:
done